Wikipedia/Code/ReadingListsFunnel.swift

```diff
@@ -16,12 +16,11 @@
     }
     
     private func event(category: EventLoggingCategory, label: EventLoggingLabel?, action: Action, measure: Int = 1) -> Dictionary<String, Any> {
-        let category = category.value
         let action = action.rawValue
         
         var event: [String: Any] = ["category": category, "action": action, "measure": measure, "primary_language": primaryLanguage(), "is_anon": isAnon]
-        if let labelValue = label?.value {
-            event["label"] = labelValue
+        if let label = label {
```

Comment:
might be good to use `.rawValue` here and the other places to make it explicitly a `String`